### PR TITLE
Added possibility to reparent a group of tracks of animation player to another node

### PR DESCRIPTION
**Pull request by AndreaCatania**
With this PR is possible to change the current node path of an animation track to another.

ezgif com-video-to-gif 1

(Sorry if I've used a broken animation to show it)

### CODE REVIEW

File: editor/animation_track_editor.cpp [hunk 1]
```diff
@@ -1429,11 +1429,11 @@ void AnimationTrackEdit::_notification(int p_what) {
 			}
 		}
 
-		if (in_group) {
-			draw_line(Vector2(timeline->get_name_limit(), get_size().height), get_size(), linecolor);
-		} else {
-			draw_line(Vector2(0, get_size().height), get_size(), linecolor);
-		}
+		//if (in_group) {
```
Comment (akien-mga) [inline editor/animation_track_editor.cpp]: Why is this commented out? If not needed it should be removed.

Comment (AndreaCatania) [reply editor/animation_track_editor.cpp]: It's just an error. I'll fix it in 10 min

**Comment by AndreaCatania** [comment]
@akien-mga Removed that comment

**Comment by reduz** [comment]
Sorry, but this PR is useless. Given how animations are imported from 3D DCCs (in a very static form), modifying them is pointless. Whathever you are trying to do, will have to be done in a different way..

**Comment by AndreaCatania** [comment]
It was useful for me when I was importing animations from Mixamo that don't give the possibility to download all animation all together and this mean that I would download these in separate files.

When I was importing these to Godot (using preset settings "with separate animation") they was no more valid due to structure of skeleton changed. So this functionality of retargeting was useful..

**Comment by reduz** [comment]
I think we should probably add some retargeting node, like SkeletonRetarget, which works similar to an existing skeleton but allows to retarget the animations...

**Comment by AndreaCatania** [comment]
Hmm, but why don't retarget it directly? Probably the problem is that the original dae will be corrupted... In that case would be useful make a copy during retargetting... Add another node to it make the Tree editor a bit crowded.

Imagine in case of more then one or two animation to retarget with different Skeleton path

**Pull request closed without merging**